contrib/builder/deb/arm/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM armhf/debian:jessie
```

Comment:
More specifically, I mean move it to the detect-os-arch script as a
"case" instead.